core/examples/http_bench_json_ops/http_bench_json_ops.js

```diff
@@ -13,27 +15,24 @@ const responseBuf = new Uint8Array(
 
 /** Listens on 0.0.0.0:4570, returns rid. */
 function listen() {
-  return Deno.core.ops.op_listen();
+  return ops.op_listen();
```

Comment:
Isn't this opening up users to prototype vulnerabilities if they follow the example then?

```
$ deno eval "Deno.core.ops.op_listen = 23;console.log(Deno.core.ops.op_listen);"
23
```